data_structures/src/serializers.rs

```diff
@@ -0,0 +1,572 @@
+        },
+    );
+
+    build_flatbuffer(builder, message)
+}
+
+// Create a witnet's ping message to decode a flatbuffers' ping message
+fn create_ping_message(ping_args: &PingWitnetArgs) -> Message {
+    Message {
+        kind: Command::Ping {
+            nonce: ping_args.nonce,
+        },
+        magic: ping_args.magic,
+    }
+}
+
+// Create a pong flatbuffer to encode a witnet's pong message
+fn create_pong_flatbuffer(
+    builder: &mut FlatBufferBuilder,
+    pong_args: &PongFlatbufferArgs,
+) -> Vec<u8> {
+    let pong_command = FlatBuffersPong::create(
+        builder,
+        &PongArgs {
+            nonce: pong_args.nonce.to_owned(),
+        },
+    );
+    let message = FlatBuffersMessage::create(
+        builder,
+        &MessageArgs {
+            magic: pong_args.magic,
+            command_type: FlatBuffersCommand::Pong,
+            command: Some(pong_command.as_union_value()),
+        },
+    );
+
+    build_flatbuffer(builder, message)
+}
+
+// Create a witnet's pong message to decode a flatbuffers' pong message
+fn create_pong_message(pong_args: &PongWitnetArgs) -> Message {
+    Message {
+        kind: Command::Pong {
+            nonce: pong_args.nonce,
+        },
+        magic: pong_args.magic,
+    }
+}
+
+// Create a verack flatbuffer to encode a witnet's verack message
+fn create_verack_flatbuffer(
+    builder: &mut FlatBufferBuilder,
+    verack_args: VerackFlatbufferArgs,
+) -> Vec<u8> {
+    let verack_command = FlatBuffersVerack::create(builder, &VerackArgs {});
+
+    let message = FlatBuffersMessage::create(
+        builder,
+        &MessageArgs {
+            magic: verack_args.magic,
+            command_type: FlatBuffersCommand::Verack,
+            command: Some(verack_command.as_union_value()),
+        },
+    );
+    build_flatbuffer(builder, message)
+}
+
+// Create a witnet's verack message to decode a flatbuffers' verack message
+fn create_verack_message(verack_args: VerackWitnetArgs) -> Message {
+    Message {
+        kind: Command::Verack,
+        magic: verack_args.magic,
+    }
+}
+
+// Create a version flatbuffer to encode a witnet's version message
+fn create_version_flatbuffer(
+    builder: &mut FlatBufferBuilder,
+    version_args: &VersionFlatbufferArgs,
+) -> Vec<u8> {
+    let sender_address_command = match version_args.sender_address.ip {
+        WitnetIpv4 { ip } => {
+            let ip_command = FlatBuffersIpv4::create(builder, &FlatBuffersIpv4Args { ip });
+            FlatBufferAddress::create(
+                builder,
+                &AddressArgs {
+                    ip_type: FlatBuffersIpAddress::Ipv4,
+                    ip: Some(ip_command.as_union_value()),
+                    port: version_args.sender_address.port,
+                },
+            )
+        }
+        WitnetIpv6 { ip0, ip1, ip2, ip3 } => {
+            let ip_command =
+                FlatBuffersIpv6::create(builder, &FlatBuffersIpv6Args { ip0, ip1, ip2, ip3 });
```

Comment:
Same thing here but with `FlatBuffersIpv6`. This code is repeated later on in this function and also it was written in the `create_peers_flatbuffer` function. I would suggest a small refactor into a function to avoid repeating code.